website/cue/reference/releases/0.32.1.cue

```diff
@@ -0,0 +1,59 @@
+package metadata
+
+releases: "0.32.1": {
+	date:     "2023-08-21"
+	codename: ""
+
+	whats_next: []
+	description: """
+		This patch release contains a fix for a regression in 0.32.0 and fixes a few issues with the release artifacts.
+
+		**Note:** Please see the release notes for [`v0.32.0`](/releases/0.32.0/) for additional changes if upgrading from
+		`v0.31.X`. In particular, the upgrade guide for breaking changes.
+		"""
+
+	changelog: [
```

Comment:
Is it worth also calling out the otel max payload size adjustment here as well?

Reply:
Thanks for catching that. I could have sworn I added a note for it, but it seems like I didn't.